TenureInformationApi.Tests/V1/E2ETests/Stories/AddPersonToTenureTests.cs

```diff
@@ -11,20 +8,20 @@ namespace TenureInformationApi.Tests.V1.E2ETests.Stories
 {
     [Story(
        AsA = "Internal Hackney user (such as a Housing Officer or Area housing Manager)",
-       IWant = "the ability to capture a new tenure",
-       SoThat = "I can create a new tenure in the system with all the relevant information")]
+       IWant = "the ability to add a person to a tenure",
+       SoThat = "I can create a link between a tenure and a person")]
     [Collection("Aws collection")]
-    public class PatchTenureTests : IDisposable
+    public class AddPersonToTenureTests : IDisposable
```

Comment:
confused by this -I'm guessing the old tests were named incorrectly?

Reply:
Yeah... some of the classes for the AddPersonToTenure code were named as if they were for general patch processing.